client/src/pages/RoomPage.tsx

```diff
@@ -105,6 +106,10 @@ function RoomPage({ name }: { name: string }): JSX.Element {
     const onKey = (e) => {
       if (e.key === "m") {
         toggleMute();
+      } else if (e.key === "s") {
+        setShowModal(true);
+      } else if (e.key === "L") {
+        history.go(0);
       } else if (e.code === "Space") {
```

Comment:
Space actually also triggers onClick when the button is focused, so might not be the good choice.
Change this to 's' for status and ',' for preferences?